.ci/deploy.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Automatic deployer script
+# This script is meant to be run on solar.coala.io!
+
+WORKDIR=/opt/solar
+
+log() {
+    echo "$(date -u "+%b %Y %T") $HOST $USER: $*"
+}
+
+update_instance() {
+    instance=$1
+    old_path="$(pwd)"
+    full_path="$WORKDIR/$instance"
+
+    if [ -f "$full_path/docker-compose.yml" ]; then
+        cd $full_path || return
+
+        log "Updating $instance ..."
+
+        docker-compose up --force-recreate --build -d
+
+        if [ $? -ne 0 ]; then
+            log "An error has occured while updating $instance!"
+        fi
+
+        cd $old_path || return
+    fi
+}
+
+update_instances() {
+    for instance in "$@"; do
+        update_instance $instance
+    done
+}
+
+log "Starting Automated Deployment ..."
+
+cd $WORKDIR || exit
+
+log "Pulling changes ..."
+git pull
+
+changed_instances=$(git diff-tree --no-commit-id --name-only -r HEAD | \
```

Comment:
A couple of problems that came into mind by doing this is the fact that some of the changes from previous commit won't affect the `changed_instances` because it only checks for HEAD.

Another thing that we can do is to store the last commit id where deployment was triggered somewhere, but is it a good idea to this in the first place or is it better to just redeploy the whole services instead?